pom.xml

```diff
@@ -56,7 +56,7 @@
       <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <version>24.1.1-jre</version>
+        <version>20.0</version>
```

Comment:
20.0 is later version supports Java 7
https://github.com/google/guava/wiki/ReleaseHistory